sparse/usr/sbin/adbd-functionfs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
Maybe "set -e" at the start so in case if fails we actually get those in some logs?